src/main/kotlin/no/nav/syfo/motebehov/ConvertLegacyMotebehovSvarFieldsHelper.kt

```diff
@@ -0,0 +1,241 @@
+
+enum class MotebehovCreatorRole {
+    ARBEIDSGIVER,
+    ARBEIDSTAKER,
+}
+
+@Component
+class ConvertLegacyMotebehovSvarFieldsHelper {
+    private val formIdentifierArbeidsgiverSvarBehov = "motebehov-arbeidsgiver-svar"
+    private val formIdentifierArbeidsgiverMeldBehov = "motebehov-arbeidsgiver-meld"
+    private val formIdentifierArbeidsgiverUnknownSvarMeldBehov = "motebehov-arbeidsgiver-unknown"
+    private val formIdentifierArbeidstakerSvarBehov = "motebehov-arbeidstaker-svar"
+    private val formIdentifierArbeidstakerMeldBehov = "motebehov-arbeidstaker-meld"
+    private val formIdentifierArbeidstakerUnknownSvarMeldBehov = "motebehov-arbeidstaker-unknown"
+
+    private val legacyFormsSemanticVersion = "0.1.0"
+
+    private val motebehovFieldIds = mapOf(
+        "svarHarBehovRadioGroupField" to "harBehovRadioGroupField",
+        "meldOnskerMoteLegacyCheckboxField" to "onskerMoteCheckboxField",
+        "onskerSykmelderDeltarCheckboxField" to "onskerSykmelderDeltarCheckboxField",
+        "begrunnelseTextField" to "begrunnelseTextField",
+    )
+
+    private val motebehovLegacyLabels = mapOf(
+        "svarArbeidsgiverHarBehovField" to "Har dere behov for et møte med NAV?",
+        "svarArbeidstakerHarBehovField" to "Har du behov for et møte med NAV og arbeidsgiveren din?",
+        "svarHarBehovRadioOptionYes" to "Ja, jeg mener det er behov for et møte",
+        "svarHarBehovRadioOptionNo" to "Nei, jeg mener det ikke er behov for et møte",
+        "meldArbeidsgiverOnskerMoteCheckbox" to "Jeg ønsker et møte med NAV og den ansatte",
+        "meldArbeidstakerOnskerMoteCheckbox" to "Jeg ønsker et møte med NAV og arbeidsgiveren min.",
+        "meldArbeidsgiverOnskerSykmelderDeltarCheckbox" to
+            "Jeg ønsker at den som sykmelder arbeidstakeren, også skal delta i møtet.",
+        "meldArbeidstakerOnskerSykmelderDeltarCheckbox" to
+            "Jeg ønsker at den som sykmelder meg, også skal delta i møtet.",
+        "begrunnelseTextField" to "Begrunnelse"
+    )
+
+    private val formFilloutOptionIds = mapOf(
+        "svarHarBehovRadioOptionYes" to "ja",
+        "svarHarBehovRadioOptionNo" to "nei"
+    )
+
+    data class ExtractedFromLegacyForklaring(
+        val actualBegrunnelse: String,
+        val onskerSykmelderDeltar: Boolean
+    )
+
+    fun createLegacyBegrunnelseTextField(
+        begrunnelseTextValue: String,
+        harMotebehov: Boolean
+    ): FilloutTextField {
+        return FilloutTextField(
+            fieldID = motebehovFieldIds["begrunnelseTextField"] ?: "",
+            fieldLabel = motebehovLegacyLabels["begrunnelseTextField"] ?: "",
+            textValue = begrunnelseTextValue,
+            wasOptional = harMotebehov
+        )
+    }
+
+    fun createLegacySvarBehovRadioGroupField(
+        harMotebehov: Boolean,
+        motebehovCreatorRole: MotebehovCreatorRole
+    ): FilloutRadioGroupField {
+        val optionIdYes = formFilloutOptionIds["svarHarBehovRadioOptionYes"]!!
```

Comment:
Litt frarådet i kotlin å bruke !!. Får vi til å type det slik at vi slipper å tvinge assertion for at det finnes?